force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
+        String helloSf = 'Hello, Salesforce!';
         // Get the length of the String        
+        Integer stringLength = helloSf.length();
+        String lengthValue = String.valueOf(stringLength);
         // Get a substring of the String of the first 5 characters
+        String first5 = helloSf.substring(0,5);
         // Convert the String to lower case
-        
+        String lowerCase = helloSf.toLowerCase();
         // Return the list with the string data
+        List <String> returnList = new List <String>();
         // In the first position add the string length
+        returnList.add(lengthValue);
         // In the second position add the substring
+        returnList.add(first5);
         // In the third position add the string in lower case
+        returnList.add(lowerCase);
 
-        return null; // Replace null with the variable you used to store the result
+        return returnList; // Replace null with the variable you used to store the result
```

Comment:
Logically this is fine. New devs often declare a lot more variables than they need.  If all you are doing is adding it to a list then it is easiest to simply add it. See my solution for an example.

```
 public static List<String> personDetails() {
        List<String> results = new List<String>();
        Person person = new Person('John Doe', 28);
        results.add(person.introduceYourself());
        results.add(person.getName());
        results.add(String.valueOf(person.getAge()));
        return results;
    }
    ```